### DISCUSSION TITLE
All players vote count placeholder

### DISCUSSION THREAD

**Discussion by Z1xus** [Zrips/CMI#5949] [Enhancement]
Add placeholder like %cmi_user_votecount% but for all users all time/monhtly

**Comment by CloudeLecaw** [comment]
There is no database for that yet. ithe daylie votecounter is also just a temporary value for the moment. Each time you restart the server the amount for the player gets resetet.
Try to vote for your server on 6 sites check the daylie votes, and do a restart the server. Check the daylie votes aggain, You will see the current placeholder for vote for the day show 0. Just the amount of total votes of a player has a database entry at the Moment.
So this daylie/weekly Check would need a extra database entry too where the votes are saved. Else you loose it after each restart. For monthly it would need an extra database entry or config value that cmi knows when it needs to get reseted.


By the way, this request was allready asked for, you find it in the discussion section.
...i know that, because it was me that requested it a half year ago ;)
